app/assets/stylesheets/_mixins.scss

```diff
@@ -10,3 +10,18 @@
   #{$property}: #{$fallback};
   #{$property}: var(--#{$cssVariable}, #{$fallback});
 }
+
```

Comment:
There is some linting going on in here as well, so the only important bits to look for are `@include themeable...`